qa/tasks/cephfs/kernel_mount.py

```diff
@@ -131,6 +131,10 @@ def umount(self, force=False):
         if not self.is_mounted():
             self.cleanup()
             return
+        if self.is_stuck():
+            self._run_umount_lf()
+            self.cleanup()
```

Comment:
Can't we do similar kind of thing in qa/tasks/cephfs/fuse_mount.py? I.e. just run `self._run_umount_lf()` and `self.cleanup()` in case of `is_stuck()`?

Reply:
https://github.com/ceph/ceph/pull/46988#discussion_r927309637

Reply:
I was talking about that condition `if cleanup:`, its not present here.

Reply:
That's because cleanup variable is also absent here.

Reply:
Okay.